components/QuestionForm.js

```diff
@@ -16,9 +16,6 @@ export default function QuestionForm({ onAddEntry }) {
 
   return (
     <Form onSubmit={handleSubmit}>
-      <StyledQuestion htmlFor="answer">
-        Worauf freust du dich heute?
-      </StyledQuestion>
       <textarea
```

Comment:
Your `textarea` is missing a `label`. I suggest leaving the label as it was beforehand. The content of the label you can change by using a prop.